#--- cctbx_sources/libtbx/env_config.py.orig	2008-09-27 19:36:05.414610789 +0200
#+++ cctbx_sources/libtbx/env_config.py	2008-09-27 19:40:56.927914755 +0200
#@@ -1028,7 +1028,7 @@
#       for file_name in test_scripts:
#         print >> f, 'libtbx.python "%s" $*' % file_name
#       f.close()
#-      os.chmod(path, 0755)
#+      # os.chmod(path, 0755)
# 
#   def pickle(self):
#     self.reset_dispatcher_support()
#--- cctbx_sources/libtbx/env_config.py.orig	2008-09-29 13:04:18.161039016 +0200
#+++ cctbx_sources/libtbx/env_config.py	2008-09-29 13:05:02.329034554 +0200
#@@ -824,7 +824,7 @@
#       print >> f, "  exec"+cmd, '"$@"'
#       print >> f, "fi"
#     f.close()
#-    os.chmod(target_file, 0755)
#+    # os.chmod(target_file, 0755)
# 
#   def windows_dispatcher(self, command_path, dispatcher_name,
#         unique_pattern="0W6I0N6D0O2W8S5_0D0I8S1P4A3T6C4H9E4R7",
--- cctbx_sources/libtbx/env_config.py	2009-01-19 08:54:57.000000000 +0100
+++ cctbx_sources/libtbx/env_config.py.new	2009-03-04 12:15:08.000000000 +0100
@@ -891,8 +891,8 @@
     else:
       action = self.write_bin_sh_dispatcher
       ext = ""
-      try: os.chmod(source_file, 0755)
-      except OSError: pass
+      #try: os.chmod(source_file, 0755)
+      #except OSError: pass
     target_file_ext = target_file + ext
     remove_or_rename(target_file_ext)
     try: action(source_file, target_file_ext)
